lib/evaluation/handlers/sameSubjectEvaluationHandler.ts

```diff
@@ -79,7 +62,7 @@ export class SameSubjectEvaluationHandler extends AbstractEvaluationHandler {
    * We have input descriptor to field ids mapping. This function gets a (reverse) map from field id to input descriptor
    */
   private findSameSubjectFieldIdsToInputDescriptorsSets() {
-    this.pDefinition.input_descriptors.forEach(this.mapFieldIdsToInputDescriptors());
+    this.pDefinition?.input_descriptors.forEach(this.mapFieldIdsToInputDescriptors());
```

Comment:
Since pDefinition can be undefined it might be handy to also do the same for input_descriptors (so add an additional ?)

Reply:
I got your point, I'll fix it.